javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/ignore/UserExcludedClassLoaderConfigurer.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.tooling.ignore;
+
+import static java.util.Collections.emptyList;
+
+import com.google.auto.service.AutoService;
+import io.opentelemetry.javaagent.extension.ignore.IgnoredTypesBuilder;
+import io.opentelemetry.javaagent.extension.ignore.IgnoredTypesConfigurer;
+import io.opentelemetry.sdk.autoconfigure.spi.ConfigProperties;
+import java.util.List;
+
+@AutoService(IgnoredTypesConfigurer.class)
+public class UserExcludedClassLoaderConfigurer implements IgnoredTypesConfigurer {
+
+  // visible for tests
+  static final String EXCLUDED_CLASSLOADER_CONFIG = "otel.javaagent.exclude-classLoader";
```

Comment:
rename variable to `EXCLUDED_CLASS_LOADERS_CONFIG`
rename flag to `otel.javaagent.exclude-class-loaders`